### ISSUE TITLE
Add "ontologies to search" to user profile

### ISSUE THREAD

**Issue by graybeal** [metadatacenter/cedar-template-editor#707]
Allow the user to describe which ontologies to search (when building templates) in their user profile.

(Note: Similar ideas referenced in https://github.com/metadatacenter/cedar-template-editor/issues/554 and https://github.com/metadatacenter/cedar-template-editor/issues/614, both since closed.)

**Comment by martinjoconnor** [comment]
Related to https://github.com/metadatacenter/cedar-project/issues/412

**Comment by martinjoconnor** [comment]
Another option is to link user profile to BioPortal slice. @marcosmro  will need to examine.

**Comment by marcosmro** [comment]
Here are some analysis notes for this task: https://docs.google.com/document/d/1_95Koea35CgYkWVlLd8F23AnqGAFDDQNnRUXdxAWlFs/edit?usp=sharing

**Comment by graybeal** [comment]
@martinjoconnor I think this original task was just about letting the user specify ontologies to choose from in a persistent way (and I think that task still needs to exist somewhere), but it has been taken over to address https://github.com/metadatacenter/cedar-project/issues/592, "Analyze how to extend the CEDAR Workbench to work with multiple BioPortal installations". Maybe you can sort it somehow?

**Comment by martinjoconnor** [comment]
Yup - posted above to this issue by mistake. Removing this from 1.6 and adding metadatacenter/cedar-project#592.
